tests/foreman/installer/test_installer.py

```diff
@@ -1791,6 +1762,7 @@ def test_installer_cap_pub_directory_accessibility(capsule_configured):
 @pytest.mark.tier1
 @pytest.mark.build_sanity
 @pytest.mark.first_sanity
+@pytest.mark.pit_client
```

Comment:
If you want to keep the `pit_client` marker here, you have to ensure that `lru_sat_ready_rhel` has a configurable non-hardcoded workflow name coming from settings that is not `settings.server.deploy_workflow`. It should ideally be `settings.content_host.rhelX.vm.workflow`. Also `deploy_rhel_version` should be configured either to `8`  - simply X version only to ensure we are not requesting the released version of RHEL instead of the non-released one. To get an insight on how pytest is configured in PIT testing, please see `automationPITPipeline.groovy`.


https://github.com/SatelliteQE/robottelo/blob/0a0966ba4a895bf91635eac4b0a7e44b2aaed92d/robottelo/hosts.py#L71-L80

Reply:
Opened #13138 for this.